scripts/README.md

```diff
@@ -14,46 +14,83 @@ npx truffle compile
 yarn run networks-inject
 ```
 
-Setup the env variables:
+Create a gnosis-safe wallet [here](https://rinkeby.gnosis-safe.io). This wallet will be called your Master Safe in the following. It is used to bundle the transactions and setup the bracket-traders.
+This Master Safe must have an additional owner `0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1`, known as the "Proposer" account. The following scripts will use this account to propose transactions to the interface. This implies that the mnemonic phrase for this "Proposer" account is stored in plain text within this project.
+
+In order to have a save setup, make sure that _your Master Safe always requires one more signature than just the signature of the Proposer account to send a transaction_. _Otherwise, everyone can steal the funds from your account!_
```

Comment:
It wouldn't be too difficult, I believe, to have a different proposer account. In this way there would be less risk of our users shooting themselves in the foot. Do you think it would be worth the effort?

Reply:
I created an issue